sql/2022/interop/backdrop.sql

```diff
@@ -0,0 +1,48 @@
+#standardSQL
```

Comment:
Do you think we should add a comment what the query is looking for?

Reply:
Will add